### ISSUE TITLE
(Just a question) mobile phones with Dimensity Chip 

### ISSUE THREAD

**Issue by bltoby** [aki65/aki65.github.io#118]
Is katago network b18 supported by Dimensity Chips ? e.g.Dimensity 9200 9300

**Comment by aki65** [comment]
No, the optimized 18b networks are not supported by Dimensity chips, since these networks use a Qualcomm-specific NPU interface. 
Unfortunately the NPU interface offered by Dimensity chips does not support all operations of KataGo's 18b network, so I can't build an optimized version for that interface.

**Comment by aki65** [comment]
The human SL network is an original 18b network, it's not optimized. Original 18b and 28b networks have always been working "fine" on dimensity chip, but they run on CPU and therefore they are very slow.

This doesn't matter with the human SL network, since it does only a few visits, but for analyzing this speed difference matters a lot.

But if you like, you can just load an original .bin.gz (don't give any .dlc or .tflite in the file selection box, just the .bin.gz) and it will work.
